### PR TITLE
Added BP_DeathmatchGameInstance to show delegate usage

### PR DESCRIPTION
**Pull request by improbable-andreaskrugersen**
https://blueprintue.com/blueprint/ut9fnue5/

### CODE REVIEW

**Comment by mironec** [comment]
Not sure if we need the on connected one...
If we want to keep it, I would let it say the full "Connected to SpatialOS successfully".
Also, have you tested what happens without Spatial Networking? I assume these just don't fire, even though the "OnConnected" event name seems pretty general.

**Comment by improbable-valentyn** [comment]
Could you take a screenshot of the event graph on the game instance?

File: Game/Config/DefaultEngine.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 GameDefaultMap=/Game/Maps/Deployments.Deployments
 EditorStartupMap=/Game/Maps/FPS-Start_Tiny.FPS-Start_Tiny
 GlobalDefaultGameMode=/Game/GameMode/BP_DeathmatchGameMode.BP_DeathmatchGameMode_C
-GameInstanceClass=/Script/SpatialGDK.SpatialGameInstance
+GameInstanceClass=/Game/GameMode/BP_DeathmatchGameInstance.BP_DeathmatchGameInstance_C
```
Comment (improbable-valentyn) [inline Game/Config/DefaultEngine.ini:5]: The game instance will be the same for different game modes, so perhaps `BP_GDKShooterGameInstance`?

**Comment by improbable-andreaskrugersen** [comment]
> Could you take a screenshot of the event graph on the game instance?

Did the link not work for you?

**Comment by improbable-andreaskrugersen** [comment]
Updated BP: https://blueprintue.com/blueprint/ut9fnue5/

**Pull request merged**